config/prow/plugins.yaml

```
@@ -147,6 +147,7 @@ lgtm:
   review_acts_as_lgtm: true
 - repos:
   - kubernetes/website
+  - kubernetes/kubernetes
```

nit:
```suggestion
  - kubernetes/kubernetes
  - kubernetes/website
```
(sorting alphabetically)

---

fixed